@copilot This PR was rejected due to the following clang-tidy error:
```
/work/tests/ttnn/unit_tests/gtests/test_multiprod_queue.cpp:35:18: error: using decl 'FloatEq' is unused [misc-unused-using-decls,-warnings-as-errors]
   35 | using ::testing::FloatEq;
      |                  ^
/work/tests/ttnn/unit_tests/gtests/test_multiprod_queue.cpp:35:18: note: remove the using
   35 | using ::testing::FloatEq;
      | ~~~~~~~~~~~~~~~~~^~~~~~~~
   36 | using ::testing::Pointwise;
```

Can you please review the files in this PR for any unused using declarations and remove them?

--- a/tests/ttnn/unit_tests/gtests/test_multiprod_queue.cpp
+++ b/tests/ttnn/unit_tests/gtests/test_multiprod_queue.cpp
@@ -32,7 +32,6 @@ namespace tt::tt_metal {
 namespace {
 
 using ::testing::Eq;
-using ::testing::FloatEq;
 using ::testing::Pointwise;
 using ::tt::tt_metal::is_device_tensor;
 

